pkg/bloomgateway/processor.go

```diff
@@ -52,7 +52,7 @@ func (p *processor) processTasks(ctx context.Context, tasks []Task) error {
 	return nil
 }
 
-func (p *processor) processTasksForDay(ctx context.Context, tenant string, day config.DayTime, tasks []Task) error {
```

Comment:
The Go Lint CI was complaining so I fixed these here even though they are unrelated to the PR